usagov/usagov-2021
# usagov-2097-alert-message-missing-svg-names: added svg and aria code …

…for screen readers


## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2097

## Description

Added svg tag and aria label code to announce the background svg image
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

1. Clear you cache (bin/drush cr)
2. Goto http://localhost/avoid-foreclosure
3. Start Voice Over on Mac (command F5) and tab through the left navigation
4. Before you get the alert component you need to use the arrow key to get to the component, hold down the "control" and "option" keys together while using the right arrow key.
5. You should hear the screen reader announce "info, region".
6. Continue to hold down the "control" and "option" keys together while using the right arrow key and you should hear "info,  image”
7. Continue to hold down the "control" and "option" keys together while using the right arrow key and you should hear  "Heading level 2, important” 
8. Continue to hold down the "control" and "option" keys together while using the right arrow key and you should hear the body text in the alert box 

<img width="1512" alt="Screenshot 2024-11-20 at 8 35 08 AM" src="https://github.com/user-attachments/assets/9fc0adca-4d4f-43e7-816b-7c0d0c1e2ce2">

<img width="1512" alt="Screenshot 2024-11-20 at 8 35 19 AM" src="https://github.com/user-attachments/assets/5f5d42e3-593e-4c24-93e7-b3f89e0a99ea">

<img width="1512" alt="Screenshot 2024-11-20 at 8 35 31 AM" src="https://github.com/user-attachments/assets/2fdf4304-a05f-499e-be8f-f45be2f5f725">


## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
